core/src/main/java/org/apache/accumulo/core/data/Key.java

```diff
@@ -657,6 +657,20 @@ public ByteSequence getRowData() {
     return new ArrayByteSequence(row);
   }
 
+  /**
+   * Writes the row ID into the given <code>ArrayByteSequence</code>. This method gives users
```

Comment:
By covering the following topics in some way, the javadoc could be a bit less ambiguous about the goals of this method. 

 * Avoids copying row data
 * Avoids allocating objects
 * Calls `{@link org.apache.accumulo.core.data.ArrayByteSequence#reset(byte[], int, int)}` which only copies references to data.

Reply:
Thought `Writes the row ID into the given <code>ArrayByteSequence</code>`  could be more specific.  Also thought the usage of the word copy in the current javadoc could be more specific.

Reply:
Agreed, done.